skimage/registration/tests/test_phase_cross_correlation.py

```diff
@@ -253,8 +253,8 @@ def test_disambiguate_2d(shift0, shift1):
             )
     np.testing.assert_equal(shift, computed_shift)
 
-
-def test_disambiguate_zero_shift():
+@pytest.mark.parametrize('disambiguate', [True, False])
```

Comment:
Considering that this code will be executed a lot, maybe it's enough to run with `disambiguate=True` here and include your check `assert(isinstance(computed_shift, np.ndarray))` to one of the tests above that already execute the function with `disambiguate=False` (which probably should be checked for in any case, as far as i can tell currently it's only done implicitely in a dtype check).

Reply:
I am not sure what you mean by "executed a lot". Are you getting at the performance of our test suite? I am happy with either option as long as the return type is tested for each `disambiguate` case.